hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -393,6 +393,23 @@ bool simplify(Env& env, const movzlq& inst, Vlabel b, size_t i) {
   });
 }
 
+bool simplify(Env& env, const popp& inst, Vlabel b, size_t i) {
+  if (!env.use_counts[inst.d0]) { decltype(inst.d0){Vreg::kInvalidReg}; }
+  if (!env.use_counts[inst.d1]) { decltype(inst.d1){Vreg::kInvalidReg}; }
```

Comment:
What's the purpose of these two lines?  It looks to me like you're just constructing temporary Vr's on the stack which get immediately destructed... have you actually verified that this has any effect?